rotkehlchen/db/upgrades/v30_v31.py

```diff
@@ -165,4 +165,18 @@ def upgrade_v30_to_v31(db: 'DBHandler') -> None:
    UNION ALL /* using union all as there can be no duplicates so no need to handle them */
    SELECT * from trades
 ;""")  # noqa: E501
+    cursor.execute("""CREATE TABLE IF NOT EXISTS history_events (
+    identifier TEXT NOT NULL PRIMARY KEY,
+    event_identifier TEXT NOT NULL,
+    sequence_index INTEGER NOT NULL,
+    timestamp INTEGER NOT NULL,
+    location TEXT NOT NULL,
+    location_label TEXT,
+    asset TEXT NOT NULL,
+    amount TEXT NOT NULL,
+    usd_value TEXT NOT NULL,
+    notes TEXT,
+    type TEXT NOT NULL,
```

Comment:
We also need to purge all kraken trades and their queried range, right? Since otherwise we would not be able to detect if a user had older insta trades?